manage.py

```diff
@@ -240,18 +240,27 @@ def add(ctx, collections, note, name, tables_only, field_counts_option, field_li
         name = f"collection_{'_'.join(str(_id) for _id in sorted(collections))}"
 
     schema = f'view_data_{name}'
+
+    db.execute('CREATE SCHEMA IF NOT EXISTS summaries')
+    db.set_search_path(["summaries"])
+    db.execute('''CREATE TABLE IF NOT EXISTS selected_collections
+                  (id SERIAL, schema TEXT NOT NULL, collection_id INTEGER NOT NULL)''')
+    db.execute('CREATE UNIQUE INDEX IF NOT EXISTS selected_collections_id ON selected_collections (id)')
```

Comment:
Do we need an index on `id`?

Reply:
To be honest, I'm not sure what we're using the `id` for here, could it be dropped entirely?

Reply:
Yes, I think it's fine to remove `id SERIAL`.